package.json

```diff
@@ -23,11 +26,18 @@
   },
   "license": "MIT",
   "dependencies": {
+    "asset-pipe-common": "^1.0.0-beta.6",
```

Comment:
we should scope and release 1.0.0 of this

Reply:
There is a task for it in jira as part of the asset-pipe cleanup task